### DISCUSSION TITLE
SSR missing tailwind classes (Solito example app)

### DISCUSSION THREAD

**Discussion by cglacet** [nativewind/nativewind#782] [Q&A]
I've tried to use the Solito example app, but its seems like there is a missing part. I also went over the documentation without finding what it was.

I only managed to have the CSS classes embeded in the `_app.js` file, it looks a bit like this: 

```js
 function(module, __webpack_exports__, __webpack_require__) {
    "use strict";
    eval(__webpack_require__.ts("__webpack_require__.r(__webpack_exports__);\n/* harmony import */ var _node_modules_next_dist_build_webpack_loaders_css_loader_src_runtime_api_js__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ../../node_modules/next/dist/build/webpack/loaders/css-loader/src/runtime/api.js ETC. things like ':is(.bg-red-500) {\\n  --tw-bg-opacity: 1;\\n  background-color: rgb(239 68 68)'"))
    /***/
}
```

It works on the client side, but on the server side the CSS is not rendered, which means the page will render without any style when javascript is disabled. I don't have the same issue if I use react-native `Stylesheet.create` or embeded styles.

In other words, this :

```tsx
import { View as ReactNativeView, Text } from 'react-native'
import { styled } from 'nativewind'

import { StyleSheet } from 'react-native';

const styles = StyleSheet.create({
  red: {backgroundColor: '#ef4444'}
});

export function HomeScreen() {
  return (
    <View className="flex-1 items-center justify-center p-3" style={styles.red}>
      <Text>Welcome to Solito.</Text>
    </View>
  )
}
``` 

Will render the expected output (JS disabled):

![image](https://github.com/marklawlor/nativewind/assets/28622118/cb85bcc2-96cb-4c09-a0b8-fc51ac46ab34)

While using nativewind:

```tsx
export function HomeScreen() {
  return (
    <View className="flex-1 items-center justify-center p-3 bg-red-500">
      <Text>Welcome to Solito.</Text>
    </View>
  )
}
```

Will render image a blank div that (again, with JS disabled):

![image](https://github.com/marklawlor/nativewind/assets/28622118/9aa7cd64-bb94-4b6b-964e-d68e9407bea7)

I found issue #276 that seems vaguely related, but If I remove `nativewind` from nextJS list of transpiled packages, I get an error:

```bash
- error /Users/kunetravel/github/solito/example-monorepos/with-tailwind/node_modules/react-native/index.js:14
import typeof AccessibilityInfo from './Libraries/Components/AccessibilityInfo/AccessibilityInfo';
^^^^^^

SyntaxError: Cannot use import statement outside a module
```

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
